warehouse/templates/packaging/detail.html

```diff
@@ -244,7 +244,8 @@ <h2 class="no-top-padding">Reverse Dependencies (TODO)</h2>
             <div class="release{{ ' release--latest' if loop.first else ' release--oldest' if loop.last }}{{ ' release--current' if is_current }}">
               <div class="release__meta">
                 {% if is_current %}
-                <p class="release__version">{{ hrelease.version }}</p>
+                <p class="release__version"><a href="{{ request.route_path('packaging.release', name=release.project.name,
+                version=hrelease.version) }}">{{ hrelease.version }}</a></p>
```

Comment:
I just found it in the Travis CI report. And it looks like it's still broken.... :(